pkg/accounting/metrics.go

```diff
@@ -52,12 +55,31 @@ func newMetrics() metrics {
 			Name:      "credit_events_count",
 			Help:      "Number of occurrences of BZZ credit events towards peers",
 		}),
-		AccountingDisconnectsCount: prometheus.NewCounter(prometheus.CounterOpts{
+		AccountingDisconnectsEnforceRefreshCount: prometheus.NewCounter(prometheus.CounterOpts{
 			Namespace: m.Namespace,
 			Subsystem: subsystem,
-			Name:      "accounting_disconnects_count",
+			Name:      "accounting_disconnects_enforce_refresh_count",
+			Help:      "Number of occurrences of peers disconnected based on failed refreshment attempts",
```

Comment:
the `accounting_` prefix is already added automatically. this will lead to `accounting_accounting_`